microsoft-edge/extensions-chromium/publish/publish-extension.md

```diff
@@ -125,57 +133,85 @@ Define the specific markets in which you plan to offer your extension.  The defa
+| **Mature content** | - | Checkbox to specify if your extension includes mature content.  Extension rating helps determine the appropriate age group of the target audience of your extension.  To help determine if your extension has mature content, see [Developer policies for the Microsoft Edge Add-ons store](/legal/microsoft-edge/extensions/developer-policies). |
 
-Select **Save & Continue** to continue to the **Store listings** section.
+Properties that are not indicated as Required are Optional.
 
-> [!IMPORTANT]
-> Your developer/organization name, website URL, and support contact details that you submitted during registration are displayed to users on the Microsoft Edge Add-ons website.
+Your developer/organization name,<!-- todo: what field name? --> **Website URL**, and **Support contact details** that you submitted<!-- todo: before, or in this form? --> during registration are displayed to users on the Microsoft Edge Add-ons website.
+
+After filling in the **Properties** webpage for the extension, click the **Save & Continue** button.
+
+The **Store listings** section opens.<!-- todo: a webpage in this context, or section of webpage (which webpage & section)? -->
 
 
 <!-- ========================================================================== -->
 ## Step 5: Add Store listing details for your extension
 
-The information provided in the following section is displayed to users who review your listing on the Microsoft Edge Add-ons website.  Even though some fields are optional, you should provide as much information as possible.  To list your extension in the store, the following details are required.
+The information provided in the **Store listings** webpage<!-- todo: a webpage in this context, or section of webpage (which webpage & section)? --> is displayed to users who view your listing on the Microsoft Edge Add-ons website.  Even though some fields are optional, you should provide as much information as possible.
+
+In the **Language details** section<!-- todo: a webpage in this context, or section of webpage (which webpage & section)? -->, fill in the following properties.
+
+| Property name | Required? | Description |
+|---|---|---|
+| **Display name** | Required for at least one language | The `name` of your extension specified in the manifest file of your extension.  To change the store display name after submission, you can update the name in the manifest file, create a new extension package, and then re-upload it. |
+| **Description** | Required for each language | The `description` field explains what your extension does, why users should install it, and any other relevant information users need to know.  Length: Minimum 250 characters, maximum 5000 characters.  Clarity: Must clearly describe the full functionality of your extension.  Completeness: Should be thorough and should leave no important details out.  Appropriateness: Must not contain any misleading or inappropriate content. |
```

Comment:
Can we make the three points into bullet points
[Clarity](

- Clarity: Must clearly describe the full functionality of your extension.
- Completeness: Should be thorough and should leave no important details out.
- Appropriateness: Must not contain any misleading or inappropriate content.)

Reply:
@Lakshmisha-KS , @bhuvanapriyap  - in Markdown, table cells are extremely limited formatting.
* bold
* links
* `<br/>` linebreak is supported, I tried it here, didn't work, will try again.  Will try * Foo.
* not lists

Reply:
a solution pattern: when too much content in a table cell, requiring formatting, change to:
simple minimal content in table cell, + section heading below the table containing details/ formatting.

Reply:
"naive" tagging test: attempting to put list within table cell:
![image](https://github.com/user-attachments/assets/70d8de7c-1ed0-4416-a6fe-5660ec20dffd)

result (per VS Code > Preview, with Learn Auth Pack installed):
![image](https://github.com/user-attachments/assets/f748f4ea-a37f-4fcb-96e9-d3e7c5c6799a)

Reply:
test 2: `<br/>* Completeness: Should be thorough and should leave no important details out.`

result: (according to VS Code, which USUALLY is ALMOST same as real rendering):
![image](https://github.com/user-attachments/assets/cfde55db-744c-4eb1-9519-24a01fbfe40e)

Reply:
test 3: concated list items: `need to know.  <br/>* Length: Min`
result: pretty good:
![image](https://github.com/user-attachments/assets/af887e26-5774-48c8-a318-745f92b2e5d8)

Reply:
test 4: restoring bold per Bhuvana's initial commit:
```
| **Description** | Required for each language | The `description` field explains what your extension does, why users should install it, and any other relevant information users need to know.  <br/>* **Length:** Minimum 250 characters, maximum 5000 characters.  <br/>* **Clarity:** Must clearly describe the full functionality of your extension.  <br/>* **Completeness:** Should be thorough and should leave no important details out.  <br/>* **Appropriateness:** Must not contain any misleading or inappropriate content. |
```

result per Preview: 
![image](https://github.com/user-attachments/assets/2fccc044-7b7d-4636-b768-7bc80420f8c8)

will push commit and check real rendered at 
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/publish-extension?branch=pr-en-us-3330#step-5-add-store-listing-details-for-your-extension

Reply:
success:
![image](https://github.com/user-attachments/assets/9f84d431-a017-4a72-9b9d-afc03cccf07b)

thx @Lakshmisha-KS